src/style.css

```diff
@@ -0,0 +1,67 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf"),
+    url("fonts/Avenir-Heavy.ttf"),
+    url("fonts/Avenir.ttc");
+  font-style: normal;
+  font-weight: 300;
+}
```

Comment:
For each font style you should add separate @font-face (one for normal style, one for bold)